app/javascript/components/public_health/ConfirmExport.js

```diff
@@ -35,7 +66,8 @@ class ConfirmExport extends React.Component {
 
 ConfirmExport.propTypes = {
   show: PropTypes.bool,
-  title: PropTypes.string,
+  exportType: PropTypes.string,
```

Comment:
As part of this prop change, you need to update the instantiation of this component in `CustomExport.js`.

Reply:
done